cli/integration/tests/waiter/test_cli.py

```diff
@@ -0,0 +1,180 @@
+import logging
+import uuid
+
+import pytest
+
+from tests.waiter import util, cli
+
+
+@pytest.mark.cli
+@pytest.mark.timeout(util.DEFAULT_TEST_TIMEOUT_SECS)
+class WaiterCliTest(util.WaiterTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.waiter_url = util.retrieve_waiter_url()
+        util.init_waiter_session(cls.waiter_url)
+
+    def setUp(self):
+        self.waiter_url = type(self).waiter_url
+        self.logger = logging.getLogger(__name__)
+
+    def test_basic_create(self):
+        token_name = self.current_name()
```

Comment:
Can we randomize + timestamp the name and log it so that we do not need the delete below (and be sure of no conflicts)?

Reply:
Why wouldn't we want to delete?

Reply:
Ah, I thought you meant the delete in the `finally`. I understand now what you meant. Will do.

Reply:
Done:

```bash
2019-03-04 22:21:47,930 [INFO] waiter --url http://localhost:9091  create test_basic_create_20190304T222147_9oBvMi1U --cmd 'RESPONSE="HTTP/1.1 200 OK\r\n\r\nOK"; while { printf "$RESPONSE"; } | nc -l "$PORT0"; do   echo "====="; done' --cpus 0.1 --mem 128 --cmd-type shell --version 43266fc1-62b9-47bb-a0e5-0f6ada49bad1
```